src/System.Linq/tests/System.Linq.Tests.csproj

```diff
@@ -79,16 +80,23 @@
     <Compile Include="$(CommonTestPath)\System\AssertExtensions.cs">
       <Link>Common\System\AssertExtensions.cs</Link>
     </Compile>
-    <Compile Include="$(CommonTestPath)\System\Diagnostics\DebuggerAttributes.cs">
-      <Link>Common\System\Diagnostics\DebuggerAttributes.cs</Link>
-    </Compile>
     <Compile Include="$(CommonTestPath)\System\PlatformDetection.cs">
       <Link>Common\System\PlatformDetection.cs</Link>
     </Compile>
     <Compile Include="$(CommonTestPath)\System\Linq\SkipTakeData.cs">
       <Link>Common\System\Linq\SkipTakeData.cs</Link>
     </Compile>
   </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetGroup)'!='uapaot'">
```

Comment:
can you add a comment for the next dev like ` tests that inherently depend
on private framework Reflection and will
never work on .Net Native. The proper way to test
these is by using the debugger expression evaluator,
not Reflection.`